packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts

```diff
@@ -11,8 +13,19 @@ export enum LoadBalancerType {
 export interface LoadBalancedServiceBaseProps {
   /**
    * The cluster where your service will be deployed
+   * You can only specify either vpc or cluster. Alternatively, you can leave both blank
+   *
+   * @default create a new cluster
```

Comment:
Set the default "value" as `-`:

```
@default - create a new cluster
```

Also, mentioned that the cluster will be created with a default VPC.